### ISSUE TITLE
Print does not work with network printers over ipp

### ISSUE THREAD

**Issue by Sommerwiesel** [ONLYOFFICE/DesktopEditors#1437]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (Arch official AUR)

### Version information

7.5.1.23

### Expected Behavior

I print a document and select my CUPS network shared printer and it prints

### Actual Behavior

The printer does not receive any kind of printing task

### Reproduction Steps

1. Open any document in ONLYOFFICE
2. STRG + P
3. Select PIXMA (ipps) shared from my raspberry
4. Print
5. Nothing happens
6. CUPS webinterface shows no tasks

### Additional information

All other apps on my Linux are able to print.
If I save the document as PDF, open it in Okular and print, it works.

I use unencrypted ipp because the printer is not shared over WAN, there is really no need for encryption in my local network and the hassle of setting up a self-signed-cert.

### DEBUG LOG

```
** (DesktopEditors:6935): WARNING **: 16:44:15.174: failed to get find a colord device: The name org.freedesktop.ColorManager was not provided by any .service files
QXcbConnection: XCB error: 8 (BadMatch), sequence: 2992, resource id: 81788991, major code: 130 (Unknown), minor code: 3
QFileInfo::absolutePath: Constructed with empty filename
QPainter::begin(): Returned false
```

**Comment by LuckySpecialist** [comment]
Hi @Sommerwiesel, It was not possible to repeat your problem.
Please specify the exact operating system and environment. It would be great if you provide a short video showing a demonstration of the problem.

**Comment by LuckySpecialist** [comment]
Hi @Sommerwiesel!
I apologize for the long answer, I returned to this problem, but the video is no longer available :(
I would like to note that we do not have an officially supported AUR package.
I checked again the printing from Ubuntu 22.04 with the DEB package versions 7.5.1 and 8.0.1 in conjunction with the printer server on raspberry pi + CUPS 2.3.3op2. I didn't find a problem.
Is the problem relevant to you on the new version of the editors from the official repository?
